10.1.0/startup/generateStartupCommand.js

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
```

Comment:
we can improve the readability and maintainability of this file by using some functions, but at a later date